polylogue/drive.py

```
@@ -24,11 +24,11 @@
 def _retry(fn, *, retries: int = 3, base_delay: float = 0.5):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_retry`

(ANN202)